deploy/Makefile

```
@@ -1,6 +1,6 @@
 JSONNET_FMT := jsonnetfmt -n 2 --max-blank-lines 2 --string-style s --comment-style s
 VERSION ?= $(shell git describe --exact-match --tags $$(git log -n1 --pretty='%h') 2>/dev/null || echo "$$(git rev-parse --abbrev-ref HEAD)-$$(git rev-parse --short HEAD)")
-SERVER_VERSION ?= $(shell curl -s https://api.github.com/repos/parca-dev/parca/releases/latest | grep -oP '"tag_name": "\K(.*)(?=")' | xargs echo -n)
+SERVER_VERSION ?= $(shell curl -s https://api.github.com/repos/parca-dev/parca/releases/latest | grep -oE '"tag_name":(.*)' | grep -o 'v[0-9.]*' | xargs echo -n)
```

Looks like a job for `jq` :)

```suggestion
SERVER_VERSION ?= $(shell curl -sSf https://api.github.com/repos/parca-dev/parca/releases/latest | jq -r .tag_name)
```

And for `curl`, `-S` ensure we output errors even with `-s`, and `-f` fails the command for non-2xx HTTP statuses